ibazel/command/command.go

```diff
@@ -16,25 +16,36 @@ package command
 
 import (
 	"bytes"
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"runtime"
 	"strings"
+	"syscall"
+	"time"
 
 	"github.com/bazelbuild/bazel-watcher/bazel"
+	"github.com/bazelbuild/bazel-watcher/ibazel/log"
 	"github.com/bazelbuild/bazel-watcher/ibazel/process_group"
 )
 
-var execCommand = process_group.Command
-var bazelNew = bazel.New
+var (
+	execCommand      = process_group.Command
+	bazelNew         = bazel.New
+	gracefulDuration = flag.Duration(
+		"graceful_duration",
```

Comment:
Nit: since this is a flag people will never have to type, only tests I'm inclined to make it painfully obvious what it is doing. WDYT about renaming it to `gracefull_termination_wait_duration`?

Reply:
This is the inner struggle of choosing between what's right and what feels good. While shorter names feel good the more descriptive names are the right thing to do. I agree!

I was thinking to make departure between variable name and flag name as the variable name does have a context so maybe it doesn't have to be that long. Maybe just `waitDuration` would be sufficient.

I don't speak english as my native language but isn't it "graceful" with one "l"? At least that's what Google tells me. :) Changing name to `graceful_termination_wait_duration` with a cautious assumption that I would be right about the spelling.